server.js

```diff
@@ -25,11 +65,57 @@ const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
-app.use(bodyParser.json())
+app.use(express.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Welcome to my mongoDB API!')
+})
+
+app.get('/members', async (req, res) => {
+  const { name } = req.query
+
+  if (name) {
+    try {
+      const allMembers = await Member.find({ 
+        name: {
+          $regex: new RegExp(name, "i")
+        } 
+      })
+      res.json(allMembers)
+    } catch (error) {
+      res.status(400).json({ error: 'Something went wrong', details: error })
+    }
+  } else {
+    try {
+      const allMembers = await Member.find()
+      res.json(allMembers)
+    } catch (error) {
+      res.status(400).json({ error: 'Something went wrong', details: error })
+    }
+  }
+})
+
+app.get('/members/:id', async (req, res) => {
+  const { id } = req.params
+
+  try {
+    const singleMember = await Member.findById(id)
+    res.json(singleMember)
+  } catch (error) {
+    res.status(400).json({ error: 'Something went wrong', details: error })
+  }
+})
+
+app.get('/members/name/:memberName', async (req, res) => {
+  const { memberName } = req.params
+
+  try {
+    const singleMember = await Member.findOne({ name: memberName })
+    res.json(singleMember)
+  } catch (error) {
+    res.status(400).json({ error: 'Something went wrong', details: error })
+  }
```

Comment:
This is probably all on my end but I can't get at member to show using https://sandra-project-mongo-api.herokuapp.com/members/name/poya
I get Null